app/components/search.tsx

```diff
@@ -171,13 +194,86 @@ const ResultItem = ({
       key={title}
       title={tooltip}
       onClick={() => onSelect(pageid, title)}
-      onMouseDown={() => setHide(false)}
+      onMouseDown={() => setHide && setHide(false)}
     >
       {title}
     </button>
   )
 }
 
+const ShowMoreSuggestions = ({
+  question,
+  relatedQuestions,
+  openQuestionTitles,
+  onClose,
+  onSelect,
+}: {
+  question: string
+  relatedQuestions: string[]
+  openQuestionTitles: Props['openQuestionTitles']
+  onSelect: Props['onSelect']
+  onClose: (e: any) => void
+}) => {
+  const title = `Does any of the following questions match your question?`
+  const [extraQuestions, setExtraQuestions] = useState<SearchResult[]>(empty)
+
+  useEffect(() => {
+    const getResults = async (question: string) => {
+      try {
+        const results = await (
+          await fetch(`/questions/search?question=${encodeURIComponent(question)}`)
+        ).json()
+        setExtraQuestions(results)
+      } catch (error) {
+        console.error(error)
+        setExtraQuestions([])
```

Comment:
what should happen if the request fails? This will result in an empty list, but I'm guessing an error should be displayed?

Reply:
The NLP endpoint will always try to return the 10 closest questions. If it fails, we can either (1) return an error or preferably (2) add the question to suggestions, let the user know and thank them.

Reply:
I went with option 2